app/bundles/CoreBundle/Helper/AssetGenerationHelper.php

```diff
@@ -33,7 +32,6 @@ class AssetGenerationHelper
         'dropzone/dist/dropzone.js', // Needed for the file upload in the asset detail page.
         'multiselect/js/jquery.multi-select.js', // Needed for the multiselect UI component.
         'chart.js/dist/Chart.js', // Needed for the charts.
-        'mousetrap/mousetrap.js',
```

Comment:
Why was mousetrap library removed?

Reply:
see https://github.com/mautic/mautic/pull/13216/commits/0f82476aded6f0c584b4f924dc6fd874adbc745d
it's dupicate